FormalConjectures/ErdosProblems/41.lean

```diff
@@ -0,0 +1,56 @@
+/-
+Copyright 2025 The Formal Conjectures Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-/
+
+import FormalConjectures.Util.ProblemImports
+
+variable {α : Type} [AddCommMonoid α]
+
+/-!
+# Erdős Problem 41
+
+*Reference:* [erdosproblems.com/41](https://www.erdosproblems.com/41)
+-/
+
+open Classical
+
+def PairCondition (A : Set α) : Prop := ∀ᵉ (I : Finset α) (J : Finset α),
+    I.toSet ⊆ A ∧ J.toSet ⊆ A ∧ I.card = 2 ∧ J.card = 2 ∧
+    (∑ i ∈ I, i = ∑ j ∈ J, j) → I = J
+
+def TripleCondition (A : Set α) : Prop := ∀ᵉ (I : Finset α) (J : Finset α),
```

Comment:
I think it would be good to have docstrings for these two!

Reply:
It may also be worth just having one definition that handles the general case (distinct `n`-term sums)